cloud/gcp/templates/gke_autopilot/main.tf

```diff
@@ -0,0 +1,187 @@
+# google_client_config and kubernetes provider must be explicitly specified like the following.
+data "google_client_config" "default" {}
+
+provider "google" {
+  project = var.project_id
+  region  = var.region
+}
+
+provider "google-beta" {
+  project = var.project_id
+  region  = var.region
+}
+
+data "google_container_cluster" "civiform_cluster" {
+  name     = var.cluster_name
+  location = var.region
+}
+
+provider "kubernetes" {
+  host                   = "https://${data.google_container_cluster.civiform_cluster.endpoint}"
+  token                  = data.google_client_config.default.access_token
+
+  // protect traffic to the KCP with TLS
```

Comment:
**suggestion:** It may be worth spelling this out and including the abbreviations in parentheses.

Reply:
Done